### DISCUSSION TITLE
spinpol calculation

### DISCUSSION THREAD

**Discussion by icamps** [grimme-lab/xtb#876] [Q&A] [answered]
Hi,

I would like to know how to implement/configure the spin polarized calculations. From the `xtb -h` command, we have this info:

```
spinpol,
    enables spin-polarization for xTB methods (tblite required)

```

But I didn't find anything else in the documentation. Any help will be highly appreciated.

Regards,

Camps

**Comment by haneug** [comment, marked as answer]
Hi Camps,
Indeed, there is not much documentation on the spin polarization and I intend to change that, but I just did not find the time to do it yet. 
The spin-polarization is available via the tblite library, which is still quite rudimentarily integrated into xtb.
You can perform a spin-polarized calculation for a triplet by running
`xtb mol.xyz --spinpol --tblite --uhf 2` 
Note that the spin polarization will only have an effect for high-spin cases. You can compare the energy with the energy of 
`xtb mol.xyz --tblite --uhf 2`
and it should be lower. 
Geometry optimization with the tblite is available but not with implicit solvation. Also the plain tblite printout is used and no printout options are available.

**Reply by icamps** [reply]
Thank you very much!
